docs/source-pytorch/advanced/training_tricks.rst

```diff
@@ -270,7 +270,12 @@ initial learning rate.
 .. warning::
 
     For the moment, this feature only works with models having a single optimizer.
-    LR Finder support for DDP and any of its variations is not implemented yet. It is coming soon.
+
+
+.. note::
+
+    With DDP since each process runs in isolation, which ever process gets the suggest learning rate first, that learning rate
+    will be broadcasted to all other processes.
```

Comment:
```suggestion
    With DDP: Since each process runs in isolation, whichever process gets the suggest learning rate first, that learning rate
    will be broadcasted to all other processes.
```